server/gokbg3/grails-app/domain/wekb/AutoUpdatePackageInfo.groovy

```diff
@@ -69,6 +73,7 @@ class AutoUpdatePackageInfo {
         countRemovedTipps (nullable:true)
         countNewTipps (nullable:true)
         onlyRowsWithLastChanged  (nullable:true)
+        kbartHasWekbFields (nullable: true,default: false)
```

Comment:
contradicts changesets 5 and 6 of changelog which prepare not null column; it is advisory to make bool columns not null anyway because binarity (true/false) is not ensured otherwise